ci/start_init_test.sh

```diff
@@ -395,7 +395,8 @@ verify_delete_replica_unsuccess() {
 verify_delete_replica() {
     old_replica_count=$(get_replica_count $CONTROLLER_IP)
     echo "$old_replica_count"
-    curl -X "POST" http://$CONTROLLER_IP:9501/v1/delete | jq
+    id=`curl http://$CONTROLLER_IP:9501/v1/volumes | jq '.data[0].id' |  tr -d '"'`
+    curl -X "DELETE" http://$CONTROLLER_IP:9501/v1/volumes/$id?action=deletevolume | jq
```

Comment:
Would it make sense to update the appropriate method in controller wo that we don't need to pass `action=deletevolume`? The Header `DELETE` becomes redundant.

Reply:
@princerachit i'm not getting you.

Reply:
Why do we need action delete when we are requesting with header DELETE?

Reply:
We generate the link for the set of actions such as revert, shutdown, start etc. So we get a link for delete action. I believe having an link for the delete action ensures we are performing right operation. Now question is what if we don't make use of action at all.

It's possible to do so, but this allows us to perform delete only with one call but above approach helps us to validate the delete url first (by generating a valid url which has been done earlier) and then perform delete.